scripts/submit_eval_jobs.py

```diff
@@ -660,7 +660,7 @@ def adjust_gpus(task_spec, experiment_group, model_name, gpu_multiplier):
     experiment_name = experiment_name.replace('β', '').replace(r"{", "").replace(r"}", "") # hack: remove characters beaker doesn't like
     d["description"] = experiment_name
     # specific image for safety eval
-    d["tasks"][0]["image"]["beaker"] = "hamishivi/open-safety"
+    d["tasks"][0]["image"]["beaker"] = "hamishivi/open_safety_1124"
```

Comment:
FYI, you can use the `use_alternate_safety_image` flag to change this. I would probably not commit this change.